pyro/contrib/oed/__init__.py

```diff
@@ -0,0 +1,6 @@
+from pyro.contrib.oed import search, eig
```

Comment:
Add a docstring with a brief description of the function of the `oed` module as the first line of this file; see e.g. `contrib/autoguide/__init__.py` for an example.